DMX.Gatekeeper.Api/Brokers/DmxApis/DmxApiBroker.Commands.cs

```diff
@@ -0,0 +1,17 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// ---------------------------------------------------------------
+
+using System.Threading.Tasks;
+using DMX.Gatekeeper.Api.Models.Commands;
+
+namespace DMX.Gatekeeper.Api.Brokers.DmxApis
+{
+    public partial class DmxApiBroker
+    {
+        private const string LabCommandsRelativeUrl = "api/labcommands";
+
+        async ValueTask PostCommand(LabCommand labCommand) =>
```

Comment:
This should be `public ValueTask<LabCommand> PostLabCommandAsync`

We have to stay consistent with the model name and the service name, namespace, exceptions and so on.